runtime/jcl/common/jclexception.c

```diff
@@ -372,3 +379,38 @@ getStackTrace(J9VMThread * vmThread, j9object_t * exceptionAddr, UDATA pruneCons
 
 	return result;
 }
+
+#if JAVA_SPEC_VERSION >= 11
+/**
+ * Set the includeClassLoaderName and includeModuleVersion fields for a StackTraceElement.
+ *
+ * @param vmThread The VM thread.
+ * @param element The element to set fields for.
+ * @param classLoader The classloader to check.
+ */
+static void
+setStackTraceElementFields(J9VMThread *vmThread, j9object_t element, J9ClassLoader *classLoader) {
+	J9JavaVM *vm = vmThread->javaVM;
+	BOOLEAN includeClassLoaderName = TRUE;
+	BOOLEAN includeModuleVersion = TRUE;
+
+	/**
+	 * If the classloader is one of the Platform or Bootstrap built-in classloaders,
+	 * don't include its name or module version in the stack trace. If it is the
+	 * Application/System built-in classloader, don't include the class name, but
+	 * include the module version.
+	 */
+	if ((NULL == classLoader)
+		|| (vm->extensionClassLoader == classLoader) // JRE: Platform ClassLoader
+		|| (vm->systemClassLoader == classLoader) // JRE: Bootstrap ClassLoader
+	) {
+		includeClassLoaderName = FALSE;
+		includeModuleVersion = FALSE;
+	} else if (vm->applicationClassLoader == classLoader) { // JRE: System ClassLoader
+		includeClassLoaderName = FALSE;
```

Comment:
@pshipton I would be casting these, e.g. `includeClassLoaderName = (BOOLEAN) FALSE`?

I'm having some issues testing in my personal builds, so just wanted to confirm before pushing here and running the win jdk11 build again.

Reply:
No, there is nothing wrong the boolean's themselves, it's when they are used to set the java field. I believe this will fix it. If you push the change I can start a PR build here.
```
J9VMJAVALANGSTACKTRACEELEMENT_SET_INCLUDECLASSLOADERNAME(vmThread, element, (U_32)includeClassLoaderName);
J9VMJAVALANGSTACKTRACEELEMENT_SET_INCLUDEMODULEVERSION(vmThread, element, (U_32)includeModuleVersion);
```

Reply:
Ah I see. I had tried that fix originally, but it looks like my personal build failed due to other reasons. I've pushed now.